### ISSUE TITLE
keep getting notifications (windows) for an update (0.25.1) but clicking it does nothing

### ISSUE THREAD

**Issue by anandijain** [KittyCAD/modeling-app#3886]
![image](https://github.com/user-attachments/assets/ea65489d-687c-4bd7-8bb7-5e4020f5c188)

i also can't seem to find out how to update from within the modeling app

**Comment by anandijain** [comment]
i just needed to restart the program. wasn't clear that was the fix though

**Comment by franknoirot** [comment]
You're absolutely right @anandijain. We just migrated desktop systems and [need to restore our custom updater dialog](https://github.com/KittyCAD/modeling-app/issues/3872) so we can show users that they're about to restart the app to update and restart it for them. I've created https://github.com/KittyCAD/modeling-app/issues/3903 to track UX improvements for finding where to update, but want to leave this open as a bug to get the team's thoughts on it.

**Comment by franknoirot** [comment]
@pierremtb I think I recall some discussion about Windows not allowing us to kill and restart the app after updating; am I fabricating that memory?
